cves/kernel/CVE-2022-36879.yml

```diff
@@ -322,8 +322,8 @@ vouch:
 
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of what your answer was.
-  answer:
-  note:
+  answer: true
+  note: there were multiple people who signed off on the commit https://github.com/torvalds/linux/commit/f85daf0e725358be78dfd208dea5fd665d8cb901
```

Comment:
It's okay to cite the exact link, but also list out the names

Reply:
Thanks! I listed the multiple people who signed off on the commit.